slides/ds101-worskhop.mdx

```diff
@@ -119,6 +119,10 @@ There are mockups of small pieces of some interface. Each piece has two copies.
 
 Participants are divided into two groups: designers and developers. Designers and developers go to two different rooms where they get their tasks. Each participant gets a mockup of some interface. Mockups do not have any additional information, just a sketch of some UI. Participants get the following 5-minute tasks:
 
+Mockups:
+
+- [1](/assets/Naming-UI-1.png)
```

Comment:
There should be a way to host images and provide links to them. But these images should not be included into slides because participants get different ones.

Reply:
I solved this in #13

Reply:
In #24 I introduced a way to use images directly from Figma so no export needed.